### PR TITLE
docs(retry): Updated docs to include how to run --retry from a npm script

### PR DESCRIPTION
**Pull request by lricoy**
To run git-cz with the --retry option from a npm script we need to do: 'npm run -- --retry'

references #132

### CODE REVIEW

**Comment by lricoy** [comment]
@jimthedev @kentcdodds Not sure what you guys think, since it is a cli, but it's just to clarify for newbies, like myself, that are not aware of the need to use double `--` to pass args

File: README.md [hunk 1]
```diff
@@ -84,9 +84,9 @@ Commitizen is great on its own, but it shines when you use it with some other am
...
 
-Please note that the retry cache may be cleared when upgrading commitizen versions, upgrading adapters, or if you delete the `commitizen.json` file in your home or temp directory.  Additionally, the commit cache uses the filesystem path of the repo, so if you move a repo or change its path, you will not be able to retry a commit. This is an edge case, but might be confusing if you have scenarios where you are moving folders that contain repos.
+Please note that the retry cache may be cleared when upgrading commitizen versions, upgrading adapters, or ifx you delete the `commitizen.json` file in your home or temp directory.  Additionally, the commit cache uses the filesystem path of the repo, so if you move a repo or change its path, you will not be able to retry a commit. This is an edge case, but might be confusing if you have scenarios where you are moving folders that contain repos.
```
Comment (kentcdodds) [inline README.md]: `ifx` should be `if`

File: README.md [hunk 2]
```diff
@@ -84,9 +84,9 @@ Commitizen is great on its own, but it shines when you use it with some other am
...
+As of version 2.7.1, you may attempt to retry the last commit using the `git cz --retry` command or `npm run git-cz -- --retry` if running from a npm script.  This can be helpful when you have tests set up to run via a git precommit hook.  In this scenario, you may have attempted a Commitizen commit, painstakingly filled out all of the commitizen fields, but your tests fail. In previous Commitizen versions, after fixing your tests, you would be forced to fill out all of the fields again. Enter the retry command. Commitizen will retry the last commit that you attempted in this repo without you needing to fill out the fields again.
```
Comment (kentcdodds) [inline README.md]: `npm run git-cz` wont work unless you have a `git-cz` script (often people will call the script `commit`). You can, however use https://www.npmjs.com/package/npm-run and do that :-) I think the wording here makes that a little unclear.

You might also mention https://www.npmjs.com/package/npm-quick-run which does not require `--` to pass args :-)

Comment (jimthedev) [inline README.md]: I agree. If you made the suggestions Kent made then we can merge this.

**Comment by lricoy** [comment]
I updated the docs to try to reflect your notes. I apologize for any grammar or other mistake in advance. Thanks for all the patience guys.

**Comment by kentcdodds** [comment]
LGTM

**Comment by jimthedev** [comment]
@lricoy Looks great. Can you please rebase to squash these commits into one? Cheers.

**Comment by lricoy** [comment]
@jimthedev done. I hope this is what you meant

**Comment by kentcdodds** [comment]
I believe so.

In the future @jimthedev, maybe you could say:

> Can you please [rebase to squash these commits into one](https://egghead.io/lessons/javascript-how-to-squash-multiple-git-commits?series=how-to-contribute-to-an-open-source-project-on-github)?

:+1:

**Comment by jimthedev** [comment]
@kentcdodds Yes. Good idea. :)

**Pull request merged**